### PR TITLE
added google analytics plugin to plugins hub

### PR DESCRIPTION
**Pull request by nikirago**
Added community developed Google Analytics Log plugin per guidelines at https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#kong-hub-contributions

### CODE REVIEW

File: app/_hub/yesinteractive/kong-log-google/index.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  #   since the 1500s.
+
+#support_url: https://github.com/yesinteractive/kong-log-google/issues
+  # (Optional) A specific URL of your own for this extension.
+  # Defaults to the url setting in your publisher profile.
+
+#source_url: https://github.com/yesinteractive/kong-log-google
+  # (Optional) If your extension is open source, provide a link to your code.
+
+#license_type: MIT
```
Comment (lena-larionova) [inline app/_hub/yesinteractive/kong-log-google/index.md:55]:
```suggestion
license_type: MIT
```

File: app/_hub/yesinteractive/kong-log-google/index.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+#      - 0.30-x
+#      - 0.29-x
+
+#########################
+# PLUGIN-ONLY SETTINGS below this line
+# If your extension is a plugin, ALL of the following lines must be completed.
+# If NOT an plugin, delete all lines up to '# BEGIN MARKDOWN CONTENT'
+
+params: # metadata about your plugin
+  name: # name of the plugin in Kong (may differ from name: above)
```
Comment (lena-larionova) [inline app/_hub/yesinteractive/kong-log-google/index.md:144]:
```suggestion
  name: kong-log-google
```

File: app/_hub/yesinteractive/kong-log-google/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  #- serverless
+  #- analytics-monitoring
+  #- transformations
+  - logging
+  #- deployment
+
+type: plugin
+
+desc: Log API transactions to Google Analytics
+description: This plugin logs your Kong gateway transactions to Google Analytics. Plugin is a modification of the Kong HTTP Log plugin.
```
Comment (lena-larionova) [inline app/_hub/yesinteractive/kong-log-google/index.md:18]:
```suggestion
description: This plugin logs your Kong gateway transactions to Google Analytics. This plugin is a modification of the [Kong HTTP Log plugin](/hub/kong-inc/http-log/).
```

File: app/_hub/yesinteractive/kong-log-google/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  community_edition:
+    compatible:
+      - 2.0-x
+      - 1.5-x
+      - 1.4-x
+      - 1.3-x
+  enterprise_edition:
+    compatible:
+      - 1.5-x
+      - 1.3-x
```
Comment (lena-larionova) [inline app/_hub/yesinteractive/kong-log-google/index.md:36]: Can you add a line break right before params? It looks like only 1.3-x is displaying in the preview, wondering if this is why.

File: app/_hub/yesinteractive/kong-log-google/index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      default: 555
+      value_in_examples: 555
+      description: Client ID. This allows you to set and identify metrics in Google Analytics by a custom client ID.
+
+# BEGIN MARKDOWN CONTENT
+---
+
+## Installation & Usage
+
+A tutorial, installation steps and further information can be found at [https://github.com/yesinteractive/kong-log-google](https://github.com/yesinteractive/kong-log-google).
```
Comment (lena-larionova) [inline app/_hub/yesinteractive/kong-log-google/index.md:85]:
```suggestion
A tutorial, installation steps, and further information can be found at [https://github.com/yesinteractive/kong-log-google](https://github.com/yesinteractive/kong-log-google).
```

File: app/_hub/yesinteractive/kong-log-google/index.md (hunk 4 above)
Comment (nikirago) [reply app/_hub/yesinteractive/kong-log-google/index.md:36]: had two param sections.... removed the one not needed. should be good to go.

**Pull request merged**